guildwars2/database.py

```diff
@@ -327,13 +327,13 @@ def check(m):
         def consolidate_duplicates(items):
             unique_items = collections.OrderedDict()
             for item in items:
-                item_tuple = item["name"], item["rarity"]
+                item_tuple = item["name"], item["rarity"], item["icon"]
```

Comment:
It's probably gonna mess with this function - it's to group items with same and rarity, they could still have different icons. Should probably try another solution. Maybe just `fetch_item()` on ids[0] would be enough

Reply:
Reversed the db function and made it into fetch_item()